pkg/asset/ignition/machine/master.go

```diff
@@ -67,7 +66,10 @@ func (a *Master) Files() []*asset.File {
 
 // Load returns the master ignitions from disk.
 func (a *Master) Load(f asset.FileFetcher) (found bool, err error) {
-	fileList := f.FetchByPattern(regexp.MustCompile(`^(master-(0|([1-9]\d*))\.ign)$`))
+	fileList, err := f.FetchByPattern("master-[0-9]*.ign")
```

Comment:
There was a discussion that this matches more than we need, e.g. it matches `master-0a.ign`, but I guess that's fine.

Reply:
> There was a discussion that this matches more than we need, e.g. it matches `master-0a.ign`...

Yeah, I waved my hands around that at the end of the 286058f78 commit message ;).